std_msgs/Header header
uint8[] data
int32 cols
int32 rows
